proto/ibc/lightclients/tendermint_attestor/v1/attestation.proto

```
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package ibc.lightclients.tendermint_attestor.v1;
+
+import "cosmos/ics23/v1/proofs.proto";
```

Buf (1.55.1)

import "cosmos/ics23/v1/proofs.proto": file does not exist

(COMPILE)